software/tools/snap_maint.c

```diff
@@ -249,33 +248,28 @@ static uint32_t unlock_action(void *handle, uint32_t addr)
 	return reg;
 }
 
-static bool decode_action(uint32_t atype)
-{
-	int i;
-	int md_size = sizeof(snap_actions)/sizeof(struct actions_tab);
-
-	for (i = 0; i < md_size; i++) {
-		if (atype == snap_actions[i].dev1) {
-			if (snap_actions[i].dev1 == snap_actions[i].dev2) {
-				VERBOSE1("%s %s\n", snap_actions[i].vendor,
-						snap_actions[i].description);
-				return true;
-			}
-		}
-	}
-	return false;
-}
-
 static void snap_decode(uint64_t reg, uint32_t level)
 {
 	uint32_t atype;
 
 	atype = (uint32_t)reg;
 	VERBOSE1("     %d     0x%8.8x     0x%8.8x  ",
 		(int)(reg >> 32ll), atype, level);
-	if (decode_action(atype))
-		return;
-	VERBOSE1("UNKNOWN Action.....\n");
```

Comment:
The automatic decoding of the action type should not be removed!

Reply:
agree...this must have been done when merging with new code just before pushing it !!!